zerver/lib/actions.py

```diff
@@ -694,14 +694,40 @@ def do_deactivate_stream(stream, log=True):
 
 def do_change_user_email(user_profile, new_email):
     # type: (UserProfile, Text) -> None
-    old_email = user_profile.email
     user_profile.email = new_email
     user_profile.save(update_fields=["email"])
 
+    payload = dict(user_id=user_profile.id,
+                   new_email=new_email)
+    send_event(dict(type='realm_user', op='update', person=payload),
+               active_user_ids(user_profile.realm))
```

Comment:
I merged sending the new event, since it means we can test using the `./manage.py change_user_email` interface.